### PR TITLE
Minicart fixes

### PR DESCRIPTION
**Pull request by rocketweb-fed**
The following issues have been identified:
1. incorrect "Edit" icon
2. wrong margin for minicart-items
3. cropped qty field and misalingment "Update" button
4. "See details" caret is absent

Example: https://www.evernote.com/shard/s55/sh/56fd1505-e39a-4852-99b8-db736612b903/ba1c0b7105302be4f26fece2a9627706

Fixed: https://www.evernote.com/shard/s55/sh/56fd1505-e39a-4852-99b8-db736612b903/ba1c0b7105302be4f26fece2a9627706 (icon sizes were fixed in #47 )

### CODE REVIEW

File: web/css/vendor/magento-ui/_forms.scss [hunk 1]
```diff
@@ -140,10 +140,11 @@ $form-element-validation__border-error: lighten($form-validation-note__color-err
 @mixin lib-form-element-number-reset() {
     &::-webkit-inner-spin-button,
     &::-webkit-outer-spin-button {
-        appearance: none;
+        -webkit-appearance: none;
```
Comment (Igloczek) [inline web/css/vendor/magento-ui/_forms.scss:145]: Autoprefixer should add here proper prefixes.

File: web/css/vendor/magento-ui/_forms.scss [hunk 2]
```diff
@@ -140,10 +140,11 @@ $form-element-validation__border-error: lighten($form-validation-note__color-err
...
         margin: 0;
     }
-    appearance: field;
+
+    -moz-appearance: textfield;
```
Comment (Igloczek) [inline web/css/vendor/magento-ui/_forms.scss]: `appearance: textfield;` - Autoprefixer.

File: Magento_Checkout/css/source/module/_minicart.scss [hunk 3]
```diff
@@ -248,6 +248,14 @@ $minicart-qty__height        : 24px;
...
             // TO DO - The selector ".abs-toggling-title" was not found.
             // @extend .abs-toggling-title !optional;
+            @include lib-icon-font(
+                $_icon-font-content: $icon-down,
+                $_icon-font-size: 28px,
+                $_icon-font-line-height: 16px,
+                $_icon-font-text-hide: false,
+                $_icon-font-position: after,
+                $_icon-font-display: block
+            );
```
Comment (Igloczek) [inline Magento_Checkout/css/source/module/_minicart.scss:258]: Plz replace with aligned variables:

``` scss
@include lib-icon-font(
    $_icon-font-content    : $icon-down,
    $_icon-font-size       : 28px,
    $_icon-font-line-height: 16px,
    $_icon-font-text-hide  : false,
    $_icon-font-position   : after,
    $_icon-font-display    : block
);
```

**Comment by rocketweb-fed** [comment]
Suggested changes made. Please review. Thanks!

**Pull request merged**